### ISSUE TITLE
NES/CBI Reporting Backend: Deselect function for platforms not working

### ISSUE THREAD

**Issue by decasadecasa** [level73/membernet#236]
not able to deselect an entry in the "Select NES/CBI" list in the back-end

**Comment by decasadecasa** [comment]
Also, some autocomplete fields are not working when you select a NES/CBI (platforms,initiatives, level of action)

**Comment by IronRunes** [comment]
To deselect a NES/CBI, one needs to click on the first entry of the list, the empty one. It will get deselected, I just ran the test. 

The autocomplete fields pull form data available in the selected grant. if the data isn't available in the grant, it won't populate the fields, but will populate the ones it has data for

**Comment by decasadecasa** [comment]
It is still not pulling all the data- in particular the PLATFORMS. I tested with https://membernet.landcoalition.org/detail/grant/728 and it pulls everything but that, as well as the level of action (we have 2 fields for region, so maybe that level of action was something additional added to these forms? I do not know). 

Is it possible to make it clear by clicking on the entry itself? I think this is more intuitive and eases the training for staff

**Comment by IronRunes** [comment]
level of action is a piece of meta data only available to M&E Reports, so it cannot be pulled
I'll double check the platforms

**Comment by IronRunes** [comment]
this is fixed on dev, let me know if it looks good and I'll pop it to prod

**Comment by decasadecasa** [comment]
please do, ill review it from there

**Comment by decasadecasa** [comment]
Yes, it seems to work on dev- please push. Can you also make the selection clear upon double click in addition to the clear all empty button/space?

**Comment by IronRunes** [comment]
that's a bt more complicated - how urgent is it now that we tackled the other stuff?

**Comment by IronRunes** [comment]
following up on this - do we still need multiple platforms for a report, or can we limit it to only 1?

**Comment by decasadecasa** [comment]
We can limit to one
